src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -330,6 +330,9 @@ private function registerSwaggerConfiguration(ContainerBuilder $container, array
         if ($config['enable_swagger_ui']) {
             $loader->load('swagger-ui.xml');
             $container->setParameter('api_platform.enable_swagger_ui', $config['enable_swagger_ui']);
+        } elseif ($config['enable_redoc_ui']) {
```

Comment:
It could be nice to be able to enable both (and select which one to use using a query parameter) don't you think?

Reply:
🤔